.github/workflows/test.yml

```diff
@@ -374,6 +374,8 @@ jobs:
         include:
           - {"target": "shard-conformance", "ha": "HA",   "gateway-mode": "local",  "ipfamily": "ipv6",      "disable-snat-multiple-gws": "snatGW",   "second-bridge": "1br", "ic": "ic-disabled"}
           - {"target": "shard-conformance", "ha": "HA",   "gateway-mode": "shared", "ipfamily": "ipv4",      "disable-snat-multiple-gws": "snatGW",   "second-bridge": "1br", "ic": "ic-disabled"}
+          - {"target": "shard-conformance", "ha": "HA",   "gateway-mode": "local",  "ipfamily": "dualstack", "disable-snat-multiple-gws": "snatGW",   "second-bridge": "1br", "ic": "ic-disabled"}
+          - {"target": "shard-conformance", "ha": "HA",   "gateway-mode": "shared", "ipfamily": "dualstack", "disable-snat-multiple-gws": "snatGW",   "second-bridge": "1br", "ic": "ic-disabled"}
```

Comment:
I think that ovn-ci-periodic ran always in shared gateway mode in HA and noHa modes, which does not match what you are replacing it with here :thinking:

Reply:
An example: https://github.com/ovn-org/ovn-kubernetes/actions/runs/5397944965
```
e2e-dual (shard-conformance, HA, dualstack)
e2e-dual (shard-conformance, noHA, dualstack)
```
I think you're right, I should change the first one to
```
- {"target": "shard-conformance", "ha": "noHA", "gateway-mode": "shared", "ipfamily": "dualstack", "disable-snat-multiple-gws": "snatGW", "second-bridge": "1br", "ic": "ic-disabled"}
```
I think.

Reply:
Done.